src/lib.rs

```diff
@@ -4458,6 +4458,20 @@ impl Connection {
         self.peer_error.as_ref()
     }
 
+    /// Returns the error [`close()`] was called with, if any.
+    ///
+    /// Note that a `Some` return value does not necessarily imply
```

Comment:
We should also note that quiche can internally close connections, so a value might be returned even if the user/app didn't call `close()` explicitly.